src/views/forms/AccountManagementFormsNew/FeeLevelTag.tsx

```diff
@@ -0,0 +1,19 @@
+import { Tag, TagSign } from '@/components/Tag';
+
+export enum FeeLevel {
+  High = 'High',
+  Low = 'Low',
+}
+
+type FeeLevelTagProps = {
+  feeLevel: FeeLevel;
+};
+
+const feeLevelToSign = {
+  [FeeLevel.High]: TagSign.Negative,
+  [FeeLevel.Low]: TagSign.Positive,
+};
+// TODO [onboarding-rewrite]: add localization
+export const FeeLevelTag = ({ feeLevel }: FeeLevelTagProps) => {
+  return <Tag sign={feeLevelToSign[feeLevel]}>{feeLevel} Fees</Tag>;
+};
```

Comment:
this feels over-abstracted. just do this? also prefer sentence casing

```suggestion
export const FeeLevelTag = ({ feeLevel }: { feeLevel: 'high' | 'low' }) => {
  if (feeLevel === 'high') {
     return <Tag sign={TagSign.Negative}>{stringGetter('High fees')}</Tag>;
  }
  
  return <Tag sign={TagSign.Positive}>{stringGetter('Low fees')}</Tag>;

};
```